docs/en/ufs/Ufs-Extension-API.md

```diff
@@ -130,18 +131,32 @@ In addition, to avoid collisions, specify scope for the dependency `alluxio-core
 </dependencies>
 ```
 
-### Test
+Build the tarball:
 
-Extend `AbstractUnderFileSystemContractTest` to test that the defined `UnderFileSystem` adheres to
-the contract between Alluxio and an under storage module. Look at the reference implementation to
-include parameters such as the working directory for the test.
+```bash
+mvn package
+```
 
-```java
-public final class DummyUnderFileSystemContractTest extends AbstractUnderFileSystemContractTest {
-    ...
-}
+### Install to Alluxio
+
+Install the tarball to Alluxio:
+
+```bash
+/bin/alluxio extensions install <path>/<to>/<probject>/target/alluxio-underfs-<ufsName>-<version>.jar
 ```
 
+### Test the Under Storage Extension
+
+To ensure the new under storage develop correctly and fulfill the minimum requirements to work with Alluxio, 
```

Comment:
nit: 
- under storage -> under storage module
- UFS tests -> contract tests to test different workflows with various combinations of operations against the under storage

Reply:
Changed, thanks